src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Server Data Encoding</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QComboBox" name="comboBox_encoding">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="toolTip">
```

Comment:
![selection_100](https://cloud.githubusercontent.com/assets/110988/26034648/06df2194-38c0-11e7-9a82-7162853af0f3.png)

Here's a tooltip that I believe achieves the goals I've set out above. Do you you think? 

I did also remove the temporary text because it's an unstated assumption that anything in the `development` branch is under development. The whole thing might or might not work and we don't support anyone running unreleased code.